ibm_catalog.json

```diff
@@ -0,0 +1,686 @@
+                "key": "cloud_logs_retention_period",
+                "options": [
+                  {
+                    "displayname": "7",
+                    "value": 7
+                  },
+                  {
+                    "displayname": "14",
+                    "value": 14
+                  },
+                  {
+                    "displayname": "30",
+                    "value": 30
+                  },
+                  {
+                    "displayname": "60",
+                    "value": 60
+                  },
+                  {
+                    "displayname": "90",
+                    "value": 90
+                  }
+                ]
+              },
+              {
+                "key": "cloud_logs_data_cos_bucket_name"
+              },
+              {
+                "key": "cloud_logs_metrics_cos_bucket_name"
+              },
+              {
+                "key": "cloud_logs_cos_buckets_class",
+                "options": [
+                  {
+                    "displayname": "Standard",
+                    "value": "standard"
+                  },
+                  {
+                    "displayname": "Smart",
+                    "value": "smart"
+                  }
+                ]
+              },
+              {
+                "key": "management_endpoint_type_for_buckets",
+                "options": [
+                  {
+                    "displayname": "Direct",
+                    "value": "direct"
+                  },
+                  {
+                    "displayname": "Private",
+                    "value": "private"
+                  },
+                  {
+                    "displayname": "Public",
+                    "value": "public"
+                  }
+                ]
+              },
+              {
+                "key": "skip_cos_kms_iam_auth_policy"
+              },
+              {
+                "key": "kms_encryption_enabled_buckets"
+              },
+              {
+                "key": "existing_kms_instance_crn"
+              },
+              {
+                "key": "existing_kms_key_crn"
+              },
+              {
+                "key": "kms_endpoint_type",
+                "options": [
+                  {
+                    "displayname": "Public",
+                    "value": "public"
+                  },
+                  {
+                    "displayname": "Private",
+                    "value": "private"
+                  }
+                ]
+              },
+              {
+                "key": "cloud_logs_cos_key_ring_name"
+              },
+              {
+                "key": "cloud_logs_cos_key_name"
+              },
+              {
+                "key": "ibmcloud_kms_api_key"
+              },
+              {
+                "key": "existing_event_notifications_instances"
+              },
+              {
+                "key": "logs_routing_tenant_regions"
```

Comment:
I think maybe force mark this as required as this is the setting consumers will need to use to enable platform logging in their account